windows-gaming-driver/src/driver/controller.rs

```diff
@@ -13,7 +13,8 @@ use futures::future;
 
 use config::{UsbId, UsbPort, UsbBinding, MachineConfig, HotKeyAction, Action};
 use util;
-use driver::clientpipe::{GaCmdOut as GaCmd, ClipboardMessage, ClipboardType, ClipboardTypes, RegisterHotKey};
+use driver::clientpipe::{GaCmdOut as GaCmd, ClipboardMessage, ClipboardType, ClipboardTypes, RegisterHotKey, Point};
+use driver::control::ControlCmdOut as ControlCmd;
```

Comment:
`GaCmdOut` as `GaCmd` is just like that because I was lazy (Remembering times when there was no `GaCmdIn`, only `GaCmd`), I never intended for this to become a pattern ;D